indexing/src/main/java/com/google/enterprise/cloudsearch/sdk/indexing/IndexingServiceImpl.java

```diff
@@ -981,6 +996,10 @@ private String getRequestMode(RequestMode requestMode) {
     return requestMode == RequestMode.UNSPECIFIED ? this.requestMode.name() : requestMode.name();
   }
 
+  private String getQueue(String queue) {
```

Comment:
the method can be static

do we want to check if queue.isEmpty() ?

Reply:
added check for queue.isEmpty().

I do not see how this could be static, as it depends on this.defaultQueue